code/script.js

```diff
@@ -30,18 +43,310 @@ const showMessage = (message, sender) => {
+        if (genderDir === 'Female director') {
+          inputWrapper.innerHTML = `
+            <select id="select"> 
+              <option value="" selected disabled>Top 3 longer Thrillers with Female directors</option>
+              <option value="Point break">Point break</option>
+              <option value="Fatto di sangue fra due uomini per causa di una vedova. Si sospettano moventi politici">Fatto di sangue fra due uomini per causa di una vedova. Si sospettano moventi politici</option>
+              <option value="Bastards">Bastards</option>
+            </select>
+          `
+      } else {
+          inputWrapper.innerHTML = `
+            <select id="select"> 
+              <option value="" selected disabled>Top 3 longer Thrillers with Male directors</option>
+              <option value="The Killing of a Sacred Deer">The Killing of a Sacred Deer</option>
+              <option value="Sangue do Meu Sangue">Sangue do Meu Sangue</option>
+              <option value="Les Diaboliques">Les Diaboliques</option>
+            </select>`
+      }
+  } else if (type === 'comedy') {
+        if (genderDir === 'Female director') {
+          inputWrapper.innerHTML = `
+            <select id="select"> 
+              <option value="" selected disabled>Top 3 longer comedies with Female directors</option>
+              <option value="Slackwars">Slackwars</option>
+              <option value="The New the Bad and the Lazy">The New the Bad and the Lazy</option>
+              <option value="Heut ist mein Tag">Heut ist mein Tag</option>
+            </select>
+          `
+      } else {
+          inputWrapper.innerHTML = `
+            <select id="select"> 
+              <option value="" selected disabled>Top 3 longer comedies with Male directors</option>
+              <option value="The Big Lebowski">The Big Lebowski</option>
+              <option value="Miss Granny">Miss Granny</option>
+              <option value="Naui gyeolhon wonjeonggi">Naui gyeolhon wonjeonggi</option>
+            </select>
+          `
+      }
+  } else {
+        if (genderDir === 'Female director') {
+          inputWrapper.innerHTML = `
+            <select id="select"> 
+              <option value="" selected disabled>Top 3 longer documentaries with Female directors</option>
+              <option value="Selma">Selma</option>
+              <option value="American Passages">American Passages</option>
+              <option value="Totally Under Control">Totally Under Control</option>
+            </select>
+          `
+      } else {
+          inputWrapper.innerHTML = `
+            <select id="select"> 
+              <option value="" selected disabled>Top 3 longer documentaries with Male directors</option>
+              <option value="Ikaros">Ikaros</option>
+              <option value="Paradise Lost">Paradise Lost</option>
+              <option value="Amy">Amy</option>
+            </select>
+          `
+      }
+  }
+  }
```

Comment:
This is a very long and a little bit hard to read function. You repeated a lot of lines of code here.
When you have structures like this it is a sign you need to think about storing your data as arrays and objects and make javascript generate innerHTML from it 🤓